### PR TITLE
reverse method homework

### PR DESCRIPTION
**Pull request by MattRodigheri**

### CODE REVIEW

File: reverse.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+count = str_chars.length                                        #saves the length of str_chars as a variable
+while count>0                                                   #loops through str_chars
+  arr.push(str_chars.pop)                                       #pops last element off str_chars and pushes it to arr
+  count-=1                                                      #counts down
+end
+string_reversed = arr.join('')                                  #joins elements of arr together into a string and saves that as 'string_reversed'
+puts "Your string reversed is '#{string_reversed}'"             #puts 'string_reversed' to user
+
+
+
```
Comment (Borgaard) [inline reverse.rb]: Thorough commenting is very good coding practice!

**Comment by Borgaard** [comment]
Will merge when you `git mv reverse.rb MattRodigheri-MattRodigheri` from `bewd_sf_10_homework` directory in terminal, then `git push origin master`. :) This will update your pull request automatically, so you won't need to create a new one.

File: MatthewRodigheri-MatthewRodigheri/MattRodigheri [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (tgaff) [inline MatthewRodigheri-MatthewRodigheri/MattRodigheri]: Hi, @MattRodigheri it looks like you unintentionally have two files here.  You can remove this one by doing `git rm MatthewRodigheri-MatthewRodigheri/MattRodigheri`.  Don't forget to `git commit` and push your change too.  The PR (pull request) will update when you push.

**Comment by Borgaard** [comment]
WoOT nice job moving files entirely with command line, Matt! :octopus:

**Pull request merged**